.gitignore

```diff
@@ -61,4 +61,7 @@ typings/
 .env
 
 # next.js build output
-.next
\ No newline at end of file
+.next
+
+
+.vscode
```

Comment:
```suggestion
.vscode

```
lol, but sure

Reply:
Should it be added for every developer global config instead??

Reply:
Removed from project level .gitignore